<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPlainTextEdit" name="pTE_port1Recieved">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>751</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_port1Close">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыть сокет</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Статус</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_port1Open">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>90</y>
      <width>281</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Открыть сокет</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lE_sendingMessage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>631</width>
      <height>26</height>
     </rect>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="frame">
     <bool>false</bool>
    </property>
    <property name="dragEnabled">
     <bool>true</bool>
    </property>
    <property name="clearButtonEnabled">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_port1Send">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Оптимальная отправка</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>0</y>
      <width>371</width>
      <height>61</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Оборудование</string>
    </property>
   </widget>
   <widget class="QLabel" name="lb_port1Status">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Закрыт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Delay">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>90</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>70</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задержка 1</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_port1Stop">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>210</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Стоп</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Delay_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>140</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_5">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Об-1 уз.</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_4">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>170</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>4</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_8">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2 уз.- SCADA</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_7">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>140</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1 уз.- SCADA</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>100</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>KeepAlive</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_3">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>140</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>3</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_1">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>80</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_6">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Об-2 уз.</string>
    </property>
   </widget>
   <widget class="QLabel" name="status_port_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>110</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_port1Send_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить в 2 узел</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pB_port1Send_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Отправить в 1 узел</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Min_Time">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Min_Time_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>480</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Min_Time_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>510</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>120</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Задержка 2</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Opti_Time">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>490</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>466</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Оптимальный узел</string>
    </property>
   </widget>
   <zorder>lE_sendingMessage</zorder>
   <zorder>pTE_port1Recieved</zorder>
   <zorder>pB_port1Close</zorder>
   <zorder>label_5</zorder>
   <zorder>pB_port1Open</zorder>
   <zorder>pB_port1Send</zorder>
   <zorder>label_12</zorder>
   <zorder>lb_port1Status</zorder>
   <zorder>Delay</zorder>
   <zorder>label</zorder>
   <zorder>pB_port1Stop</zorder>
   <zorder>Delay_2</zorder>
   <zorder>status_port_5</zorder>
   <zorder>status_port_4</zorder>
   <zorder>status_port_8</zorder>
   <zorder>status_port_7</zorder>
   <zorder>pushButton</zorder>
   <zorder>status_port_3</zorder>
   <zorder>status_port_1</zorder>
   <zorder>status_port_6</zorder>
   <zorder>status_port_2</zorder>
   <zorder>pB_port1Send_2</zorder>
   <zorder>pB_port1Send_3</zorder>
   <zorder>Min_Time</zorder>
   <zorder>Min_Time_3</zorder>
   <zorder>Min_Time_4</zorder>
   <zorder>label_2</zorder>
   <zorder>Opti_Time</zorder>
   <zorder>label_3</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
